### PR TITLE
Refactor YouTube component to use ChakraUI

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
Migrate YouTube from styled-components to ChakraUI

## Related Issue
#6374

### CODE REVIEW

File: src/components/YouTube.tsx [hunk 1]
```diff
@@ -27,7 +22,7 @@ const YouTube: React.FC<IProps> = ({ id, start = "0", title = "YouTube" }) => {
   const baseUrl = "https://www.youtube.com/embed/"
   const src = baseUrl + id + startQuery
   return (
-    <Figure>
+    <Box as="figure" display="block" marginY="1rem">
```
Comment (pettinarip) [inline src/components/YouTube.tsx:25]:
```suggestion
    <Box as="figure" my={4}>
```
- `figure` is by default `block` displayed, we can ignore it
- there is a short version of `marginY` => `my`
- lets try to use the chakra spacing scale. 1 rem = 4 (more about this [here](https://chakra-ui.com/docs/styled-system/theme#spacing))

File: src/components/YouTube.tsx [hunk 2]
```diff
@@ -27,7 +22,7 @@ const YouTube: React.FC<IProps> = ({ id, start = "0", title = "YouTube" }) => {
...
       <iframe
```
Comment (minimalsm) [inline src/components/YouTube.tsx:26]: I don't think we'd want to refactor the iframe to Chakra. LMK if you disagree

File: src/components/YouTube.tsx (hunk 1 above)
Comment (minimalsm) [inline src/components/YouTube.tsx:25]: Keeping display block is necessary for Crowdin formatting reasons.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Good first Chakra PR @minimalsm 🚀 I've pointed out only a few improvements.

File: src/components/YouTube.tsx (hunk 2 above)
Comment (pettinarip) [reply src/components/YouTube.tsx:26]: No, thats ok 👍🏼

File: src/components/YouTube.tsx (hunk 1 above)
Comment (pettinarip) [reply src/components/YouTube.tsx]: Yes, we are going to use the default values. In the Design System too.

**Review by pettinarip — APPROVED** [review]
🚀

**Pull request merged**